pkg/sql/logictest/testdata/logic_test/partial_index

```diff
@@ -455,3 +455,41 @@ SELECT * FROM d@f_b_s_foo_idx WHERE b AND s = 'foo'
 100  110  100  foo  true
 300  330  300  foo  true
 400  400  400  foo  true
+
+#### Backfill a partial index.
+
+statement ok
+CREATE TABLE e (a INT, b INT)
```

Comment:
unfortunately, we have two codepaths of how this all runs. One like this, and then one that kicks in when doing schema changes on a new table created in the same transaction. You should pretty much duplicate your tests with those cases.

Reply:
Ahh good to know. Done.